spec/requests/api/v1/users/show_spec.rb

```diff
@@ -1,17 +1,24 @@
-describe 'GET api/v1/users/:id', type: :request do
-  let(:user) { create(:user) }
+# frozen_string_literal: true
+
+describe 'GET api/v1/users/:id' do
   subject { get api_v1_user_path, headers: auth_headers, as: :json }
 
+  let(:user) { create(:user) }
+
   it_behaves_like 'there must not be a Set-Cookie in Header'
 
```

Comment:
```suggestion
before { subject }
```

And remove the subjects from each unit test

Reply:
I think in this case doesn't work right? because in one of our test we have an allow_instance before the subject